Rubberduck.VBEEditor/VBEHost/CorelDRAWApp.cs

```diff
@@ -0,0 +1,17 @@
+using Corel.GraphicsSuite.Interop.CorelDRAW;
+
+namespace Rubberduck.VBEditor.VBEHost
+{
+    public class CorelDRAWApp : HostApplicationBase<Application>
+    {
+        public CorelDRAWApp() : base("CorelDRAW") { }
+
+		//Function RunMacro(ModuleName As String, MacroName As String, Parameter() As Variant) As Variant
+		//Where ModuleName appears to mean ProjectName, and MacroName has to be provided as "ModuleName.ProcName"
+		
+        public override void Run(QualifiedMemberName qualifiedMemberName)
+        {
+            Application.GMSManager.RunMacro(qualifiedMemberName.QualifiedModuleName.ProjectName.ToString(), qualifiedMemberName.ToString(), null);
```

Comment:
If the 3rd parameter is `params` (`ParamArray` in VBA), then it should be omitted; passing a `null` there amounts to passing a `Nothing` parameter value to a parameterless procedure - not going to work.